### DISCUSSION TITLE
Bug: "Convert to Issue" button does not show up in Firefox

### DISCUSSION THREAD

**Discussion by nvie** [community/community#5643] [Projects and Issues]
Hey, I just noticed that in GitHub Issues task list, hovering a task row does not make the "Convert to Issue" button show up in Firefox. If I use Chrome on the same project, same issue, it _does_ show up.

---

✅ In Chrome:
![Screen Shot 2021-09-14 at 13 14 07@2x](https://user-images.githubusercontent.com/83844/133247834-16585960-74f1-4123-a0b0-4552cbec8b03.png)

❌ In Firefox:
![Screen Shot 2021-09-14 at 13 14 18@2x](https://user-images.githubusercontent.com/83844/133247839-ccd37c6f-f292-40b9-954e-ffa30821e526.png)

**Comment by ADTC** [comment]
I'd consider this to be a feature. I'm glad I use Firefox. Please don't fix this 🙃

**Comment by navanjr** [comment]
The "Convert to Issue" button is also not appearing in Chrome 111

**Comment by yamin8000** [comment]
It's funny that it's working for issues created a while ago but for issues created recently "convert to issue" is disappeared.

**Reply by navanjr** [reply]
In my experience it works in one `org` and not in another. works in one `repo` and not in another. across my projects its "mostly" working.

**Reply by yamin8000** [reply]
> In my experience it works in one `org` and not in another. works in one `repo` and not in another. across my projects its "mostly" working.

This is very frustrating.
